__tests__/utils/utilities.spec.ts

```diff
@@ -1,10 +1,25 @@
 import { getRuntimeId, isNotEmpty } from "../../src/utils/utilities";
+import { isClientIdNotFoundErrorMessage, isCustomerMessage, isSystemMessage } from "../../src/utils/utilities";
+
+/**
+ * @jest-environment node
+ */
 
 /* eslint-disable @typescript-eslint/no-var-requires */
 const { MessageType } = require("../../src");
-const utilities = require('../../src/utils/utilities');
 
 describe('Utilities', () => {
+
+    beforeEach(() => {
+        jest.resetModules();
+    }
```

Comment:
nit: `}` & `);` formatting

Reply:
Can we get a task to enable VSCode autofix lint rules so we don't need manual review for this? I can share examples of repos with this setup elsewhere if needed as well.

Reply:
I have added extra rules and ensure those rules are enforced when building